fst/en/inflection.py

```diff
@@ -0,0 +1,60 @@
+"""
+Inflects English nouns - creates a plural form from singular.
+
+Run from project root folder.
+
+Install Pynini package before running.
+"""
+
+import os
+import pynini as p
+
+from pynini.lib import pynutil
+from pynini.lib import rewrite
+
+def _priority_union(q: p.Fst, r: p.Fst, sigma: p.Fst) -> p.Fst:
+  complement_domain_q = sigma - p.project(q, 'input')
+  return p.union(q, complement_domain_q @ r)
+
+_v = p.union('a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U')
+_c = p.union('b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n',
+             'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z',
+             'B', 'C', 'D', 'F', 'G', 'H', 'J', 'K', 'L', 'M', 'N',
+             'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'X', 'Y', 'Z')
```

Comment:
Can we replace this by a UnicodeSet and Unicode normalization to handle all of the vowels versus consonants?

Personally I split the vowel set in English to be with a y when it's a suffix, and as a consonant as a prefix.